core/infra/server/http_server_with_cappuccino.cpp

```diff
@@ -74,7 +75,26 @@ namespace http {
 
     void server() {
         logger::info("server") << "initialize server!";
-        Cappuccino::Cappuccino( 0, nullptr);
+
+        // well, I know this is ugly but I had to pass char* (without const)
+        std::string param1 = "";
+        char *param1_c = new char[param1.length() + 1];
+        strcpy(param1_c, param1.c_str());
+
+        std::string param2 = "-p";
+        char *param2_c = new char[param2.length() + 1];
+        strcpy(param2_c, param2.c_str());
+
+        std::string port = std::to_string(config::IrohaConfigManager::getInstance().getHttpPortNumber(1204));
+        char *port_c = new char[port.length() + 1];
+        strcpy(port_c, port.c_str());
+
+        char *params[] = {param1_c, param2_c, port_c};
```

Comment:
As far as I know this way is not safe in the case if a constless string be modified.